src/CommunityToolkit.Maui.Core/Layouts/UniformItemsLayoutManager.shared.cs

```diff
@@ -30,10 +30,15 @@ public override Size ArrangeChildren(Rect rectangle)
 		var width = rectangle.Width - uniformItemsLayout.Padding.HorizontalThickness;
 		var visibleChildren = uniformItemsLayout.Where(x => x.Visibility == Visibility.Visible).ToArray();
 
+		if (visibleChildren.Length == 0 || width == 0)
+		{
+			return rectangle.Size;
+		}
+
 		var columns = GetColumnsCount(visibleChildren.Length, width);
 		var rows = GetRowsCount(visibleChildren.Length, columns);
 
-		var boundsWidth = columns == 0 ? 0 : width / columns;
+		var boundsWidth = width / columns;
```

Comment:
It looks like this line of code should never execute if `GetColumnsCount(visibleChildren.Length, width) == 0`. 

Can you confirm this is accurate? 

My concern is throwing a divide by zero exception

Reply:
I added this 2 checks above if (visibleChildren.Length == 0 || width == 0). So I don't expect divide by zero here. at least our sample app and tests are passed.